### DISCUSSION TITLE
Volume for Single File

### DISCUSSION THREAD

**Discussion by kaaax0815** [caprover/caprover#1126] [Q&A] [answered]
How can I make a Volume for a single file?

**Comment by githubsaturn** [comment, marked as answer]
It's not a good practice. But if you do need it, you can create a file on your host and use a host path to map the file to a location inside your container

**Reply by kaaax0815** [reply]
And if the Location inside the Container should be a File?

**Reply by githubsaturn** [reply]
I haven't tested it, but it shouldn't be any different. What's the issue that you're facing?
